ocsp/ocsp.go

```diff
@@ -0,0 +1,82 @@
+// Package ocsp handles responding to ocsp requests and saving new ones
+package ocsp
+
+import (
+	"github.com/levenlabs/locksmith/config"
+	"log"
+
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/cloudflare/cfssl/ocsp"
+	goocsp "golang.org/x/crypto/ocsp"
+	"net/http"
+	"os"
+)
+
+var responder *ocsp.Responder
+
+func init() {
+	if config.OCSPRespFile == "" {
+		return
+	}
+
+	src, err := ocsp.NewSourceFromFile(config.OCSPRespFile)
+	if err != nil {
+		log.Fatalf("Failed to read ocsp responses file: %s", err)
+	}
+
+	log.Printf("Registering OCSP responder handler")
```

Comment:
It's kind of weird to have this here, it'd be a lot cleaner to keep all the http handler stuff together, maybe having a method in here which just returns an `http.Handler`. Or at least have a comment near the `ListenAndServe` stating all the places where other handlers are being made.